### PR TITLE
Tests for obfuscated bundles

### PR DESCRIPTION
**Pull request by arielsilvestri**
Introduces unit and integration tests for the bundle obfuscation feature.

### CODE REVIEW

File: qa/__tests__/smoke.test.ts [hunk 1]
```diff
@@ -137,5 +137,19 @@ describe('Smoke Tests', () => {
     expect(classicCookies['ajs_user_id']).toContain('Test')
 
     compareSchema(results)
+
+    const [url2, chrome2] = await Promise.all([server(true), browser()])
```
Comment (pooyaj) [inline qa/__tests__/smoke.test.ts:141]: Should we extract this out into its own test? feels like a separate test with new setup etc.

File: qa/lib/runner.ts [hunk 2]
```diff
@@ -34,18 +33,9 @@ export async function run(params: ComparisonParams) {
...
-          const obfuscatedUrl = request
-            .url()
-            .replace(new RegExp(key, 'g'), obfuscatedKey)
-          bundleRequests.push(request.url())
-          bundleRequests.push(obfuscatedUrl)
-        } else {
-          bundleRequests.push(request.url())
+        bundleRequests.push(request.url())
+        if (request.url().includes('next-integration')) {
+          bundleRequestFailures.push(request.failure())
```
Comment (pooyaj) [inline qa/lib/runner.ts:38]: This is cool 👌👌

**Review by pooyaj — APPROVED** [review]
Looks great! One minor comment otherwise lgtm

File: qa/__tests__/smoke.test.ts (hunk 1 above)
Comment (arielsilvestri) [reply qa/__tests__/smoke.test.ts]: It worked this time /shrug

**Pull request merged**